src/Native/System.Security.Cryptography.Native/pal_ssl.cpp

```diff
@@ -423,6 +471,7 @@ extern "C" int32_t GetSslConnectionInfo(SSL* ssl, CipherAlgorithmType* dataCiphe
     *keyExchangeAlg = MapExchangeAlgorithmType(cipher);
     *dataHashAlg = MapHashAlgorithmType(cipher);
     *dataKeySize = cipher->alg_bits;
+    *hashKeySize = GetHashKeySize(cipher);
```

Comment:
I don't see this check in the latest code. All that is needed is to change line 459 from
```
if (!ssl || !dataCipherAlg || !keyExchangeAlg || !dataHashAlg || !dataKeySize)
```
to
```
if (!ssl || !dataCipherAlg || !keyExchangeAlg || !dataHashAlg || !dataKeySize || !hashKeySize)
```